xsdata/writer.py

```diff
@@ -20,29 +20,29 @@ class CodeWriter:
     def formats(self):
         return list(self.generators.keys())
 
-    def register_generator(self, name, renderer: AbstractGenerator):
-        self.generators[name] = renderer
+    def register_format(self, name, generator: AbstractGenerator):
+        self.generators[name] = generator
 
-    def get_renderer(self, name) -> AbstractGenerator:
+    def get_format(self, name) -> AbstractGenerator:
         if name in self.generators:
             return self.generators[name]
         raise CodeWriterError(f"Unknown code generator `{name}`")
 
-    def write(self, schema: Schema, classes: List[Class], package: str, renderer: str):
-        engine = self.get_renderer(renderer)
+    def write(self, schema: Schema, classes: List[Class], package: str, format: str):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'format'](https://app.codacy.com/manual/tefra/xsdata/pullRequest?prid=5137871)